torch/_inductor/mkldnn.py

```diff
@@ -283,52 +283,75 @@ def forward(self, input):
 
 
 class LinearUnary(nn.Linear):
-    def __init__(
-        self,
-        linear: nn.Module,
-        unary: nn.Module,
-    ):
+    def __init__(self, linear: nn.Module, unary: Optional[nn.Module], input_size: list):
```

Comment:
`unary` is optional now. So does it intend to support a single `Linear`?

Reply:
Yes, you are right.